### PR TITLE
interpolateParams=true: client side prepared statements

### PR DESCRIPTION
**Pull request by shlomi-noach**
Storyline: https://github.com/github/gh-ost/issues/454

In an attempt to speed up applying of binlog events, this is a low hanging fruit: set `interpolateParams=true` ([doc](https://github.com/go-sql-driver/mysql#interpolateparams))

This will set client-side prepared statements. The benefit to that is reduced round trips to the MySQL server: only one per query instead of three per query.

### CODE REVIEW

File: go/mysql/connection.go [hunk 1]
```diff
@@ -56,5 +56,5 @@ func (this *ConnectionConfig) GetDBUri(databaseName string) string {
 		// Wrap IPv6 literals in square brackets
 		hostname = fmt.Sprintf("[%s]", hostname)
 	}
-	return fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?charset=utf8mb4,utf8,latin1", this.User, this.Password, hostname, this.Key.Port, databaseName)
```
Comment (arthurnn) [inline go/mysql/connection.go]: Do we want to add unit tests to the db uri method?

Comment (shlomi-noach) [inline go/mysql/connection.go]: @arthurnn do you have something in mind? I'm happy for any and every test addition.

Comment (arthurnn) [inline go/mysql/connection.go]: Let's merge that, I can work on these tests in a separate PR.

Comment (shlomi-noach) [inline go/mysql/connection.go]: This will only be merged after a few exhaustive days of testing in production (which is happening right now 😄 ).

**Comment by shlomi-noach** [comment]
This is being tested for `8` days now in production with no issues. Will be merged shortly.

**Comment by shlomi-noach** [comment]
ok, tested for `15` days straight, no problems.

**Pull request merged**